src/components/Review.jsx

```diff
@@ -29,6 +34,9 @@ const Review = () => {
 
 	const handleSubmit = (e) => {
 		e.preventDefault();
+		const comment = { author: user.username, body: e.target[0].value };
+		postComment(review_id, comment);
+		console.log(review_id, comment);
```

Comment:
This console.log should be removed. You also need to consider what you should do with the comment that comes back to show the user that it's been added. Currently, there's no way to know if a comment has been successfully posted.

Equally, there is no error handling here, and a failed post causes the app to crash with an unhandled promise rejection. Consider what you as a user would expect to happen in a post was unsuccessful. What kind of message would be useful?